frontend/packages/kubevirt-plugin/integration-tests/tests/vm.cloud-init.scenario.ts

```diff
@@ -0,0 +1,73 @@
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { withResource } from '@console/shared/src/test-utils/utils';
+import { get } from 'lodash';
```

Comment:
nit: please move this import to the top, followed by imports from `@console/...`, then from `../whatever` and then from `./` at last.

Reply:
Updated order of imports